config/environments/development.rb

```diff
@@ -13,8 +13,23 @@
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
 
-  # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  # actionmailer configuration
+
+  Rails.application.configure do
+    config.action_mailer.delivery_method = :smtp
+    config.action_mailer.raise_delivery_errors = true
+    config.action_mailer.perform_deliveries = true
+    config.action_mailer.smtp_settings = {
+        address: ENV["MAIL_ADDRESS"],
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.